internal/util/render/executor.go

```diff
@@ -405,6 +405,30 @@ func (pn *pkgNode) runMutators(ctx context.Context, hctx *hydrationContext, inpu
 	}
 
 	for i, mutator := range mutators {
+		if pl.Mutators[i].ConfigPath != "" {
+			// kpt v1.0.0-beta15+ onwards, functionConfigs are included in the
+			// function inputs during `render` and as a result, they can be
+			// mutated during the `render`.
+			// So functionConfigs needs be updated in the FunctionRunner instance
+			// before every run.
+			for _, r := range input {
+				pkgPath, err := pkg.GetPkgPathAnnotation(r)
+				if err != nil {
+					return nil, err
+				}
+				currPath, _, err := kioutil.GetFileAnnotations(r)
+				if err != nil {
+					return nil, err
+				}
+				if pkgPath == pn.pkg.UniquePath.String() && // resource belong to current package
+					currPath == pl.Mutators[i].ConfigPath { // configPath matches
+					fmt.Printf("\n%s\n", r.MustString())
```

Comment:
Is this print statement supposed to be here?

Reply:
No, I added it during debugging the issue and forgot to remove it. Just removed it. Thanks for point it out.